### DISCUSSION TITLE
List all directories with content

### DISCUSSION THREAD

**Discussion by ahoiroman** [nuxt/content#1949] [Q&A]
Hello there,

given this directory structure:

```
chapters
- folder-1
 - index.md 
 - _dir.yml
- folder-2
 - bar.md
```

I'd like to get a sort of "overview over all folders containing contents".

In this case the result should be:

- `folder-1`
- `folder-2`

But not `index.md`, `bar.md` or `_dir.yml`.

As I am using the document driven approach, I was thinking about using this approach:

```
<ContentList path="/chapters" v-slot="{ list }">
    <div v-for="article in list" :key="article._path">
        {{article.title}}
    </div>
</ContentList>
```

But I did not find anything about a possibility to filter partials and limit depth.

Any help is highly appreciated :-)

**Comment by svirmasalo** [comment]
My solution was to first query all the paths that contain '/cities' and then filter out those that have two slashes in them 😌 This would give a list of sub folders that can be looped without <ContentList>-component
```javascript
const cityQuery = await queryContent({
    where: {
        _path: {$contains: '/cities'},
    }
}).find();

// Filter out non-cities and paths that are not direct children of root
let cityList = cityQuery.filter((item) => {
    return item._path.split('/').length === 2;
});
```
```vue
<div v-for="city in cityList" :key="city._path">
    {{city.title}}
</div>
```
